src/display/canvas.js

```diff
@@ -863,6 +865,7 @@ function resetCtxToDefault(ctx, foregroundColor) {
     ctx.setLineDash([]);
     ctx.lineDashOffset = 0;
   }
+  ctx.filter = "none";
```

Comment:
Similar to below, should we ignore this in Node to prevent any possible issues?
```suggestion
 if (!isNodeJS) {
   ctx.filter = "none";
 }
```